pkg/cli/alpha/generate.go

```diff
@@ -64,15 +63,12 @@ If no output directory is provided, the current working directory will be cleane
 			}
 		},
 	}
+	generateCmd.Flags().StringVar(&opts.InputDir, "input-dir", "",
+		"Specifies the full path to a Kubebuilder project file. If not provided, "+
+			"the current working directory is used.")
+	generateCmd.Flags().StringVar(&opts.OutputDir, "output-dir", "",
+		"Specifies the full path where the scaffolded files will be output. "+
+			"Defaults to a directory within the current working directory.")
```

Comment:
Can you please revert those changes?
It is not related to the context of this PR